### ISSUE TITLE
Trying to get attachment file name

### ISSUE THREAD

**Issue by oneillp111** [mattgwwalker/msg-extractor#36]
I have the following code to read the msg file and get the sender name and the attachment file name but the `attach` is always `None`.  The `sender` works perfectly. 

```
import ExtractMsg
import glob`
for message in glob.glob('C:\Desktop\EMAILS\email_test\*.msg'):
         msg = ExtractMsg.Message(message)
         attach = msg._getStringStream('__substg1.0_3707')
         sender = msg._getStringStream('__substg1.0_0C1F')
         print attach
         print sender
```

**Comment by m00dimus** [comment]
The Attachment class should probably have a get filename method but the class will return filename variables defined in init.

```
import ExtractMsg
import glob

inputdir=r"C:\Desktop\EMAILS\email_test\*.msg"

for message in glob.glob(inputdir):
    msg = ExtractMsg.Message(message)
    
    for attachment in msg.attachments:
        attname_l = attachment.longFilename
        attname_s = attachment.shortFilename
    
        print("Longfilename: %s" % attname_l)
        print("Shortfilename: %s" % attname_s)
```

**Comment by TheElementalOfCreation** [comment]
> The Attachment class should probably have a get filename method but the class will return filename variables defined in init.
> 
> ```
> import ExtractMsg
> import glob
> 
> inputdir=r"C:\Desktop\EMAILS\email_test\*.msg"
> 
> for message in glob.glob(inputdir):
>     msg = ExtractMsg.Message(message)
>     
>     for attachment in msg.attachments:
>         attname_l = attachment.longFilename
>         attname_s = attachment.shortFilename
>     
>         print("Longfilename: %s" % attname_l)
>         print("Shortfilename: %s" % attname_s)
> ```

Should be fixed in version 0.18

`Attachment` class now has `longFilename`, `shortFilename`, and `cid`/`content_id` properties

**Comment by TheElementalOfCreation** [comment]
Should this issue still be present, or should it reoccur, please re-open the issue.
